BUILD.md

```diff
@@ -263,26 +234,6 @@ bin/web run
 
 The web server will be running on `localhost:8084`.
 
-Note: by default, this requires a `public-api` server and assumes that it is
-running in a Kubernetes cluster. If that isn't the case, you have some other
-options:
-
-#### 1. Connect to `public-api` locally
-
-```bash
-bin/go-run controller/cmd/public-api
```

Comment:
This approach is technically still possible, but this isn't the right command for running public-api in the foreground. You would first need to setup a port-forward to the prometheus pod, and then you'd need to pass in your kube config file with the `-kubeconfig` flag. That felt like too much of a specialized use case, however, so I opted to just remove it.